Code/GraphMol/test_fixtures.h

```diff
@@ -1,3 +1,13 @@
+//
+//  Copyright (C) 2023 Greg Landrum and other RDKit contributors
```

Comment:
I think you were the original author of this code, but we can merge it this way way if you don't want to change it.

Reply:
Oh, I'm okay with being one of the "other RDKit contributors", no need to mention me by name :)